swift-sdk/Internal/IterableActionRunner.swift

```diff
@@ -77,8 +77,7 @@ struct IterableActionRunner {
     private static func shouldOpenUrl(url: URL,
                                       from source: IterableActionSource,
                                       withAllowedProtocols allowedProtocols: [String]) -> Bool {
-        if source == .push || source == .inApp,
-           let scheme = url.scheme,
+        if let scheme = url.scheme,
```

Comment:
Not required for approval, but I think it would look cleaner using `guard`

Reply:
I think normally `guard` statements are for early exits or short circuiting code, so that  exceptions don't up the logic of the method   body. Here since the is no method body as such I  think if is better.

Reply:
In other words, `guard` is primarily to make sure that certain conditions are met before executing deeper into the method.